app/presenters/trainee_contact_details.rb

```diff
@@ -0,0 +1,40 @@
+class TraineeContactDetails
+  attr_reader :trainee
+
+  def initialize(trainee)
+    @trainee = trainee
+  end
+
+  def call
+    relevant_attributes.map { |k, v|
+      [I18n.t("presenters.TraineeContactDetails.attributes.#{k}"), v]
```

Comment:
https://github.com/DFE-Digital/register-trainee-teacher-data/blob/a206eee5d0c2cb5dd9c5dac257b6c410cea49968/app/presenters/trainee_course_details.rb#L10

I feel that is the nicer approach

https://apidock.com/rails/v5.2.3/ActiveModel/Translation/human_attribute_name

Especially given the that the override is 

```yml
        phone: "Phone number"
```

Food for thought, maybe the model should be 
`phone_number`

Reply:
Good shout, think will probably address in a separate PR once we start cleaning up the schema 👍